css/mobile.css

```diff
@@ -58,34 +65,340 @@ a {
 }
+
+main {
+  max-width: 667px;
+}
+
+#works {
+  padding: 114px 24px;
+  display: grid;
+  gap: 88px;
+}
+
+.card {
+  background-color: #fff;
+  display: flex;
+  padding: 16px;
+  flex-direction: column;
+  gap: 12px;
+  border-radius: 16px;
+  border: 1px solid #dfe1e6;
+}
+
+.card img {
+  width: 100%;
+  height: auto;
+  object-fit: cover;
+}
+
+.card h2 {
+  color: #172b4d;
+  font-size: 32px;
+  font-weight: 700;
+  line-height: 44px;
+}
+
+.primary-text {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 0;
+  gap: 30px;
+  font-weight: 600;
+  line-height: 16px;
+  height: 16px;
+  color: #344563;
+}
+
+.primary-text::marker {
+  font-size: 80px;
+}
+
+form * {
+  border-radius: 8px;
+  border: 1px solid #cfd8dc;
+  background: #fff;
+  width: 100%;
+}
+
+.primary-text > *:not(:first-child) {
+  list-style: disc;
+  color: #7a869a;
+  font-size: 13px;
+}
+
+.primary-text > *:not(:first-child)::marker {
+  font-size: 18px;
+  color: #c1c7d0;
+}
+
+.paragraph {
+  color: #344563;
+  font-size: 15px;
+  font-weight: 400;
+  line-height: 24px;
+  margin: 8px 0;
+}
+
+.tag {
+  display: flex;
+  gap: 8px;
+}
+
+button {
+  width: fit-content;
+  margin: 12px 0;
+  padding: 12px;
+  font-size: 17px;
+  font-weight: 500;
+  line-height: 24px;
+  letter-spacing: 0.51px;
+  color: #396df2;
+  background-color: transparent;
+  border-radius: 8px;
+  border: 1px solid #6070ff;
+  transition: background-color 0.3s ease;
+}
+
+#about {
+  display: flex;
+  flex-direction: column;
+  background-color: #fff;
+  border-top-right-radius: 80px;
+  padding: 114px 24px;
+  gap: 12px;
+}
+
+.social-icon-2 {
+  display: flex;
+  align-items: center;
+  gap: 24px;
+  padding-left: 10px;
+  margin-top: 16px;
+}
+
+button:hover {
+  background-color: #6070ff;
+  color: #fff;
+  cursor: pointer;
+}
+
+button:active {
+  background-color: #00f;
+}
+
+.header {
+  color: #172b4d;
+  font-size: 40px;
+  font-weight: 700;
+  line-height: 52px;
+}
+
+#tech {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+}
+
+.tech {
+  padding: 24px 12px;
+  color: #000;
+  font-size: 20px;
+  font-weight: 500;
+  line-height: 24px;
+}
+
+.underline::after {
+  content: url("../assets/arrow-F.svg");
+  display: inline-block;
+  margin-right: 8px;
+  position: absolute;
+  right: 32px;
+}
+
+.underline {
+  border-bottom: 1px solid #0000001e;
+}
+
+.dropdown::after {
+  content: url("../assets/arrow-F.svg");
+  display: inline-block;
+  margin-right: 8px;
+  position: absolute;
+  right: 32px;
+  transform: rotate(90deg);
+}
+
+#contact-form {
+  display: flex;
+  flex-direction: column;
+  gap: 12px;
+  align-items: center;
+  background-color: #6070ff;
+  border-top-left-radius: 80px;
+  padding: 103px 12px 48px 12px;
+  background-image: url(../assets/contact-form-shapes-mobile.svg);
+  background-repeat: no-repeat;
+  background-position-x: 200px;
+  background-position-y: 20px;
+  background-size: 61%;
+}
```

Comment:
- [ ] Great job implementing the changes suggested by the previous reviewer. However, Your` background-image` still needs some attention as the previous reviewer mentioned. Your `background-image` doesn't match that of the Figma design. See below for more details:

<table>
<tr>
<th align=“middle”>Figma design</th>
<th align=“middle”>Your design</th>
</tr>
<tr>
<td>
<img width="311" alt="Screenshot 2023-07-11 at 19 45 49" src="https://github.com/sagieramos/Portfolio/assets/107192076/ffa87a17-c08b-49b8-ae21-3f51458b3069">
</td>
<td>
<img width="281" alt="Screenshot 2023-07-11 at 21 47 36" src="https://github.com/sagieramos/Portfolio/assets/107192076/04a0f73a-1a60-4c10-a7c6-0b56b951f310">
</td>
</tr>
</table>

Reply:
I made all the neccessary changes that the previous reviewer required. You had view it on a different screen size which is causing differences in background positioning. In fact, I don't have my contact form showing like the one you've shown here. below is what it looks on my screen
![Screenshot from 2023-07-12 00-02-01](https://github.com/sagieramos/Portfolio/assets/31627384/3136d44e-f4eb-4e3e-a002-ca193c99fa1f)